gcloud/monitoring/query.py

```diff
@@ -637,7 +637,9 @@ def _build_label_filter(category, *args, **kwargs):
             continue
 
         suffix = None
-        if key.endswith('_prefix') or key.endswith('_suffix'):
+        ends = ['_prefix', '_suffix', '_greater', '_greaterequal',
+                '_less', '_lessequal']
+        if key.endswith(tuple(ends)):
```

Comment:
In the spirit of perfectionism:

- Please write the allowed suffixes as a literal tuple, so there is no need to convert the list to a tuple.
- Please consider placing that literal tuple in-line in the `if` condition. If that doesn't work for you, please change `ends` to `endings`.